src/robusta/core/reporting/holmes.py

```diff
@@ -55,3 +70,23 @@ def __init__(
             holmes_result=holmes_result,
             **kwargs,
         )
+
+
+class HolmesChatResult(BaseModel):
+    analysis: Optional[str] = None
+    tool_calls: Optional[List[ToolCallResult]] = None
+    conversation_history: Optional[List[dict]] = None
+
+
+class HolmesChatResultsBlock(BaseBlock):
+    holmes_result: Optional[HolmesChatResult]
+
+    def __init__(
```

Comment:
why do we need the init function here? Aren't we getting it for free from Pydantic?